### ISSUE TITLE
SDK crashed with iOS 11

### ISSUE THREAD

**Issue by bucaLee** [aws/amazon-chime-sdk-ios#511]
Hello everyone, I don't have a real device that has iOS 11 but when I test my app on browserstack with a phone that has iOS 11, it crashed at the beginning (I couldn't reach the crash logs, for this reason, I don't know why SDK crashed). This issue occurred with `0.21.2` version. But before that version, I used `0.16.5`, and there was no problem with running on iOS 11.  Did SDK cut the support in a release after 0.16.5, or is this a bug?

**Comment by bucaLee** [comment]
It looks like, SDK supports iOS11 when I inspect the framework settings. Because of that, I've changed the title of the issue.

![Uploading Screen Shot 2022-08-11 at 11.44.38.png…]()

**Comment by linsang21** [comment]
Hi @bucaLee, while I'm trying to reproduce the crash, do you mind share more information by fill in the questionnaire for bug report? 
**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Logs**
If applicable, add logs from console to help explain your problem.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Test environment Info (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Version AmazonChimeSDK: [e.g. 0.4.0]
 - Version AmazonChimeSDKMedia: [e.g. 0.4.0]
 - Can you reproduce this in the demo app?
If you are reporting a crash:
 - Are you using bitcode supported binaries?
 - Please provide full crash logs in the form of a ".crash" or ".ips" file with a binary images section.

**Additional context**
Add any other context about the problem here.

**Comment by linsang21** [comment]
I was able to reproduce the crash with iPhone 6 (iOS 11.3), got the crash log indicating it's something related to loading Network framework:
```
Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Termination Description: DYLD, Library not loaded: /System/Library/Frameworks/Network.framework/Network | Referenced from: /private/var/containers/Bundle/Application/FFBB4122-C32E-4A67-9D1A-F442BFB9D03A/AmazonChimeSDKDemo.app/Frameworks/AmazonChimeSDKMedia.framework/AmazonChimeSDKMedia | Reason: image not found
Triggered by Thread:  0
```

Looking for any suspicious change caused crash.

**Comment by ziyiz-amzn** [comment]
We are working on this issue, will update when the fix is ready to release.

**Comment by ziyiz-amzn** [comment]
The fix has been released in [v0.22.3](https://github.com/aws/amazon-chime-sdk-ios/releases/tag/v0.22.3) @bucaLee
